index.bs

```diff
@@ -195,6 +195,8 @@ namespace console { // but see namespace object requirements below
   void table(any tabularData, optional sequence&lt;DOMString> properties);
   void trace(any... data);
   void warn(any... data);
+  void dir(any data);
```

Comment:
Should we perhaps use a different variable name for single arguments, leaving "data" for `any...` cases? I'm not sure I have any good suggestions though. Maybe `datum`? Meh...